cmd/erasure-server-pool.go

```diff
@@ -42,10 +42,15 @@ import (
 type erasureServerPools struct {
 	GatewayUnsupported
 
-	serverPools []*erasureSets
+	poolMetaMutex sync.RWMutex
+	poolMeta      poolMeta
+	serverPools   []*erasureSets
 
-	// Shut down async operations
+	// Shutdown async operations
```

Comment:
```suggestion
	// Shut down async operations
```

> When you must choose between one or the other, think about that space between the words shut and down. Then imagine adding a word in that space: shutting something down. That might help you remember that the two-word option is a phrasal verb.

> Don’t let bad grammar shut down your ability to communicate the message!

:D

Source: https://www.patrickkphillips.com/grammar/shut-down-or-shutdown-should-it-be-one-word-or-two/